testbed/tests/test_images.py

```diff
@@ -130,3 +130,31 @@ async def test_save(app):
                 # suite is run; and the files have a PID in them to ensure that
                 # they're unique across test runs.
                 pass
+
+
+async def test_native_image(app):
+    # Generate an image using pillow
```

Comment:
As I've noted in previous reviews, *every* test should start with a 1 line docstring that describes the purpose of the test.

Reply:
Done.